### PR TITLE
feat: added rules page

### PR DESCRIPTION
**Pull request by partha8**

### CODE REVIEW

**Review by Dev-AkshatGupta — APPROVED** [review]
This seems great to be merged

File: styles/rules.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    .rules{
+        width: 80vw;
+    }
+}
+
+.list-rules {
+  margin: 10px 0;
+  font-size: 1.2rem;
+}
+
```
Comment (ParthParolekar) [inline styles/rules.css]: Try not to write media queries in between your css file. Keep it either at the top or the bottom.

**Review by ParthParolekar — APPROVED** [review]
Everything else looks great!

File: styles/rules.css (hunk 1 above)
Comment (partha8) [reply styles/rules.css]: done

**Pull request merged**